crates/ruff/src/rules/wemake_python_styleguide/rules/too_short_name.rs

```diff
@@ -0,0 +1,156 @@
+use std::path::Path;
+
+use ruff_text_size::TextRange;
+use rustpython_parser::ast::{self, Ranged};
+
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::checkers::ast::Checker;
+use crate::settings::Settings;
+
+use crate::rules::wemake_python_styleguide::helpers::naming;
+
+/// ## What it does
+/// Checks for short variable or module names.
+///
+/// ## Why is this bad?
+/// It is hard to understand what the variable means and why it is used, if its name is too short.
```

Comment:
I think it's worth explaining how we measure the length of a name. Is it the number of characters or the visual width (character width)?

Reply:
Good point.